static/templates/admin_user_group_list.handlebars

```diff
@@ -1,10 +1,11 @@
 {{#with user_group}}
 <div class="user-group white-box" id="{{id}}">
     <div class="alert user-group-status"></div>
-    <h4>
-        <span class="name" data-placeholder="{{t 'Name' }}" contenteditable="true" spellcheck="false">{{name}}</span>
-        <span class="spacer">—</span>
-        <span class="description" data-placeholder="{{t 'Description' }}" contenteditable="true">{{description}}</span>
+    <h4 class="stream-name">
+        <span class="name" data-placeholder="{{t 'Name' }}" contenteditable="false" spellcheck="false">{{name}}</span>
+        <span class="fa fa-pencil pencil name-pencil"></span>
+        <span class="discard-changes name-discard">x</span>
+        <span class="checkmark name-check">✓</span>
```

Comment:
You want to add structure similar to stream settings in template `static/templates/subscription_settings.handlebars`. That way you can make use of existing functions.